public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -160,10 +160,10 @@ describe('LokiDatasource', () => {
       // and applyTemplateVariables is a convenient place to do that.
       const spy = jest.spyOn(ds, 'applyTemplateVariables');
 
-      const options = getQueryOptions<LokiQuery>({
+      const options = {
         targets: [{ expr: '{a="b"}', refId: 'B', maxLines: queryMaxLines }],
         app: app ?? CoreApp.Dashboard,
-      });
+      } as DataQueryRequest<LokiQuery>;
```

Comment:
We could easily do something like:

```typescript
const baseOptions = {
  requestId: '',
  interval: '',
  intervalMs: 1,
  range: getDefaultTimeRange(),
  scopedVars: {},
  timezone: '',
  app: '',
  startTime: 1,
}
const options: DataQueryRequest<LokiQuery> = {
  ...baseOptions,
  targets: [{ expr: '{a="b"}', refId: 'B', maxLines: queryMaxLines }],
  app: app ?? CoreApp.Dashboard,
};
```

So that we don't need to force the type.

Reply:
Great idea, updated in https://github.com/grafana/grafana/pull/80747/commits/7979e9a469c27b28fbfe4efa6e282d8c97cdd71b